### PR TITLE
[9.99.x-prod] SRVLOGIC-219: Update Operator's manifests to reflect devmode and builder images

### PR DESCRIPTION
**Pull request by domhanak**
Fixes use of latest and incorrect image in example dockerfile use.

Superseeds #23 as one commit was not pulled into it from fork for some reason

### CODE REVIEW

**Review by rgdoliveira — APPROVED** [review]

**Review by jakubschwan — APPROVED** [review]

File: container-builder/examples/dockerfiles/SonataFlow.dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-rhel8-operator:1.32.0 AS builder
+FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-swf-builder-rhel8:1.32.0 AS builder
```
Comment (ricardozanini) [inline container-builder/examples/dockerfiles/SonataFlow.dockerfile:18]: This doesn't matter at all since it's not used, but to test and readme.

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
Can you please fix the CI?

**Comment by rgdoliveira** [comment]
CI is failing because we removed the  go.work files on 9.99.x-prod branch to be able to build on OSBS with cachito. Can't we just ignore the CI failures as we have been doing so far?

**Comment by domhanak** [comment]
+1 we discussed this during https://github.com/kiegroup/kogito-serverless-operator/pull/22 I believe

**Pull request merged**